include/votca/xtp/matrixfreeoperator.h

```diff
@@ -23,8 +23,26 @@
 namespace votca {
 namespace xtp {
 
-class MatrixFreeOperator : public Eigen::EigenBase<Eigen::MatrixXd> {
+class MatrixFreeOperator;
+}
+}  // namespace votca
+namespace Eigen {
+namespace internal {
+// MatrixReplacement looks-like a SparseMatrix, so let's inherits its traits:
```

Comment:
can you remove that comment or at least make it correct ?
It's not a SparseMatrix